tests/integration/tests/test_util/tags.py

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright 2024 Canonical, Ltd.
+#
+from pytest_tagging import combine_tags
+
+PULL_REQUEST = "pull_request"
+NIGHTLY = "nightly"
+WEEKLY = "weekly"
+
+# Those tags can be used for a convenient way to run multiple test levels.
+combine_tags("up_to_nightly", PULL_REQUEST, NIGHTLY)
```

Comment:
Do we also need to have an `UP_TO_NIGHTLY` const? I'm not sure how it's gonna be used, but I imagine it will be _something like_:
```python
@with_tag(SOME_TAG)
def my_test(...):
    ...
```

Reply:
This is only a convenience for invoking the tests via tox.
`tox run -e integration -- --tags up_to_nightly`
A test only gets assigned one of PULL_REQUEST, NIGHTLY, WEEKLY